var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -109,6 +106,16 @@ class Libfabric(AutotoolsPackage):
              sha256='3b78d0ca1b223ff21b7f5b3627e67e358e3c18b700f86b017e2233fee7e88c2e',
              placement='fabtests', when='@1.4.2')
 
+    conflicts('@1.9.0', when='platform=darwin',
```

Comment:
What about older versions?

Reply:
From the discussions it looked like this was the first release to break, and the newer rc works fine...